src/code.jl

```diff
@@ -146,12 +146,12 @@ function function_to_expr(op::typeof(^), O, st)
             return toexpr(Term(inv, Any[ex]), st)
         else
             args = Any[Term(inv, Any[ex]), -args[2]]
-            op = get(st.rewrites, :nanmath, false) ? op : NaNMath.pow
+            op = get(st.rewrites, :nanmath, false) ? op : pow
```

Comment:
This seems wrong (was introduced in #671), shouldn't it be
```suggestion
            op = get(st.rewrites, :nanmath, false) === true ? pow : op
```